components/raftstore/src/store/peer.rs

```diff
@@ -1585,7 +1582,7 @@ where
                 // network partition from the new leader.
                 // For lease safety during leader transfer, transit `leader_lease`
                 // to suspect.
-                self.leader_lease.suspect(monotonic_raw_now());
+                self.leader_lease.suspect(now);
```

Comment:
Why change this to use `now` above?

Reply:
I thought they are CPU operations to handling these messages and should be very fast. So, it is not quite necessary to use different `now` for different messages and it should reduce some unncessary cost...

The change is not necessary, I revert it to still get a new time for each message.

Reply:
Make sense, maybe we could optimize this seperately.